### PR TITLE
RHINENG-14677: fix filters for incidents

### PR DESCRIPTION
**Pull request by Fewwy**
Fixed filters not working. You can find recordings of the filters fix in the ticket below
https://issues.redhat.com/browse/RHINENG-14677

### CODE REVIEW

File: web/src/components/Incidents/processIncidents.ts [hunk 1]
```diff
@@ -62,17 +57,15 @@ export function processIncidents(data: Incident[]): ProcessedIncident[] {
...
-        valueDate.getUTCFullYear() === currentYear &&
-        valueDate.getUTCHours() === currentHour
-      );
-    });
+    const currentDate = new Date(); // Current timestamp in milliseconds
+
+    // Calculate the time difference in milliseconds
+    const timeDifference = currentDate.valueOf() - lastDate * 1000;
+    const inactive = timeDifference < 10 * 60 * 1000;
+    const dayDifference = (lastDate - firstDate) / (60 * 60 * 24);
```
Comment (zhuje) [inline web/src/components/Incidents/processIncidents.ts:65]: Should this be (1000 * 60 * 60 * 24)?

File: web/src/components/Incidents/processIncidents.ts [hunk 2]
```diff
@@ -62,17 +57,15 @@ export function processIncidents(data: Incident[]): ProcessedIncident[] {
...
-        valueDate.getUTCMonth() === currentMonth &&
-        valueDate.getUTCFullYear() === currentYear &&
-        valueDate.getUTCHours() === currentHour
-      );
-    });
+    const currentDate = new Date(); // Current timestamp in milliseconds
+
+    // Calculate the time difference in milliseconds
+    const timeDifference = currentDate.valueOf() - lastDate * 1000;
+    const inactive = timeDifference < 10 * 60 * 1000;
```
Comment (zhuje) [inline web/src/components/Incidents/processIncidents.ts:64]: So the incident is considered inactive in 10 * 60 * 1000 = 600,000 ms = 10 minutes? Is that correct?

Comment (Fewwy) [reply web/src/components/Incidents/processIncidents.ts:64]: Yes

File: web/src/components/Incidents/processIncidents.ts (hunk 1 above)
Comment (Fewwy) [reply web/src/components/Incidents/processIncidents.ts:65]: Yes

Comment (Fewwy) [reply web/src/components/Incidents/processIncidents.ts:65]: No, the last date and first date are already converted beforehand. They

Comment (zhuje) [reply web/src/components/Incidents/processIncidents.ts:65]: Sounds good.

**Pull request merged**